src/main/resources/application.conf

```diff
@@ -1,8 +1,39 @@
 akka.http.host-connection-pool.max-open-requests: 128
 
 centaur {
-  # URL of the Cromwell to interact with
-  cromwellUrl: "http://localhost:8000"
+
+  cromwell {
+    # Cromwell configuration.
+    # Choose between 
+    # "url" : points centaur to a pre-existing Cromwell server
+    # "jar":
+    #   "path": existing cromwell jar file
+    #   "conf": existing cromwell configuration to be used 
+    #   "log": path of the log file where cromwell logs will be written
+    #   "withRestart": if true, Cromwell will be restarted when a specific test goes to Running state
+    mode: "url"
+    
+    # URL of the Cromwell to interact with
+    url: "http://localhost:8000"
+
+    # jar {
+    #   path: "/path/to/cromwell.jar"
+    #   conf: "/path/to/cromwell.conf"
+    #   log: "cromwell-log-server.log"
+    #   withRestart: true
+    # }
+    
+    # If running in jar mode, this allows to specify a second set of configuration to be used when cromwell is restarted
+    # it can be used to test migrations to some extent. If omitted, the same jar configuration will be used for restart
+    # Note that the log paths can be similar in which case logs after restart will be appended to it
+    # post-restart-jar {
+    #   path: "path/to/cromwell-jar-after-restart"
+    #   conf: "path/to/cromwell-conf-after-restart"
+    #   log: "cromwell-server.log"
```

Comment:
the example log name might be chosen to make it more clear it's a post-restart log